qiskit/pulse/library/waveform.py

```diff
@@ -97,6 +97,11 @@ def _clip(self, samples: np.ndarray, epsilon: float = 1e-7) -> np.ndarray:
     def is_parameterized(self) -> bool:
         return False
 
+    @property
+    def parameters(self) -> Dict[str, Any]:
+        """Return a dictionary containing the pulse's parameters."""
+        return dict()
```

Comment:
Why do we return an empty dict?

Reply:
Because waveform doesn't have parameters. Actually this property doesn't exist before, but I need to define this to equally handle `ParametricPulse` and `Waveform` objects in some logic.